File: livingrimoire start here/LivinGrimoire swift/DLC/Personality.swift
//
//  Personality.swift
//  LivinGrimoireSwiftV1
//
//  Created by moti barski on 11/08/2025.
//

import Foundation


func loadPersonality(_ brain: Brain) {
    brain.addSkill(DiHelloWorld())
    brain.addSkill(DiTime())
    brain.addSkill(DiSysOut())
    // brain.chained(DiHelloWorld()).chained(DiTime()).chained(DiSysOut())
}

File: livingrimoire start here/LivinGrimoire java/Personality.java
import LivinGrimoirePacket.LivinGrimoire.Brain;
import LivinGrimoirePacket.LivinGrimoire.DiHelloWorld;
import LivinGrimoirePacket.LivinGrimoire.DiSysOut;
import DLC.logical.DiStartUp;
import DLC.logical.DiTime;

public class Personality {
    public void skillsPush(Brain brain){
        brain.addSkill(new DiSysOut());
        brain.addSkill(new DiHelloWorld());
        brain.addSkill(new DiTime());
        brain.addSkill(new DiStartUp(brain));
    }
}
